corehq/apps/app_manager/tests/test_advanced_suite.py

```diff
@@ -257,10 +257,10 @@ def test_tiered_select_with_advanced_module_as_parent(self, *args):
 
         self.assertXmlPartialEqual(self.get_xml('advanced_module_parent'), app.create_suite(), "./entry[1]")
 
-    def test_tiered_select_with_advanced_module_as_parent_null_relationship(self, *args):
```

Comment:
Is it intentional to no longer be testing the advanced module?

Reply:
Yeah, it wasn't necessary and it shouldn't matter. The test indicates as if it only works with advanced module.